pages/api/submit.ts

```diff
@@ -81,7 +81,7 @@ const callLambda = async (formID: string, fields: Responses, language: string) =
       logMessage.info("Submission Lambda Client successfully triggered");
     }
   } catch (err) {
-    logMessage.error(err);
+    logMessage.error(err as Error);
```

Comment:
that should be inferred based on the catch here no ?

Reply:
My TS linter was flagging it locally as a 'Warning'.  Wanted to quiet it.. lol